--- configure.in.orig	2011-04-16 15:27:49.624886971 +0200
+++ configure.in	2011-04-16 15:29:49.391886971 +0200
@@ -116,20 +116,18 @@ fi
 
 # Check for doxygen
 
-AC_ARG_ENABLE( doxygen, [  --disable-doxygen     Disable doxygen documentation.] , disable_doxygen=yes)
+AC_ARG_ENABLE( doxygen, [  --disable-doxygen     Disable doxygen documentation.])
 
-if test "x$disable_doxygen" != "xyes" ; then
-
-AC_PATH_PROG(DOXYGEN, doxygen, no)
-
-if test x$DOXYGEN = xno; then
-   echo "Cannot find doxygen, will not build sourcecode documentation"
-fi
-
-else
-   echo "Doxygen documentation disabled, not checking"
-   DOXYGEN=no
-fi
+AS_IF([test "x$enable_doxygen" != "xno"],
+[AC_PATH_PROG(DOXYGEN, doxygen, no)],
+[echo "Doxygen documentation disabled, not checking"
+DOXYGEN=no])
+
+AS_IF([test "x$DOXYGEN" = "xno"],
+[AS_IF(
+  [test "x$enable_doxygen" = "xyes"],[AC_MSG_ERROR([Cannot find doxygen, make sure it is installed in your PATH])],
+  [test "x$enable_doxygen" = "x"],[echo "Cannot find doxygen, will not build sourcecode documentation"])
+])
 
 AM_CONDITIONAL(HAVE_DOXYGEN, test x$DOXYGEN != xno)
